lib/util/utils.js

```diff
@@ -21,5 +25,46 @@ exports.matchMemberExpression = function (name, node) {
       return true;
     }
   }
+
   return match(node.object, nameParts.length - 2);
 };
+
+/**
+ * Matches a MemberExpression by traversing the node and returns true if any of the property names match.
+ * @param nameRegex
+ * @param node
+ * @returns {Boolean}
+ */
+exports.matchRegexMemberExpression = function matchRegexMemberExpression(nameRegex, node) {
+  if (!node) {
+    return false;
+  }
+
+  if (node.type === 'MemberExpression') {
+    if (nameRegex.test(node.property.name)) {
+      return true;
+    }
+
+    return matchRegexMemberExpression(nameRegex, node.object);
+  }
+
+  return !!(node.type === 'Identifier' && nameRegex.test(nameRegex));
```

Comment:
Change this to a `if` statement for clarity.

And return false by default at the end of the function.